src/azure-cli/azure/cli/command_modules/backup/_params.py

```diff
@@ -71,6 +72,11 @@
 target_tier_type = CLIArgumentType(help=target_tier_help, arg_type=get_enum_type(allowed_target_tier_type_chk_archivable), options_list=['--target-tier'])
 tier_type = CLIArgumentType(help=tier_help, arg_type=get_enum_type(allowed_tier_type), options_list=['--tier'])
 rehyd_priority_type = CLIArgumentType(help=rehyd_priority_type_help, arg_type=get_enum_type(allowed_rehyd_priority_type), options_list=['--rehydration-priority'])
+user_assigned_identity_id_type = CLIArgumentType(options_list=['--identity-id'], help="UserAssigned Identity Id to be used for CMK encryption, this will be applicable for encryption using userassigned identity")
```

Comment:
Could we refer to the recommendation in the [guideline.md#enable-managed-identity-during-resource-creation](https://github.com/Azure/azure-cli/blob/dev/doc/managed_identity_command_guideline.md#enable-managed-identity-during-resource-creation), change the parameter name to `--mi-user-assigned`

Reply:
@zhoxing-ms : we don't have creation time experience for managed identities. This parameter is used for Customer managed key encryption properties for Vault

Reply:
Yes, because this parameter is the uniform name for adding user-assigned identities when creating and updating resources.

Reply:
Actually, this parameter `--mi-user-assigned` is the uniform name for adding user-assigned identities when creating and updating resources.
Do you mean that the parameter `--identity-id` has been used in other similar places?

Reply:
backup vault encryption update --encryption-key-id {key1_id} --identity-id {identity1_id} -g {rg} -v {vault1}

we are using this parameter for the encryption group

Reply:
OK, I see. It is still recommended to use `--mi-user-assigned` in line with the guideline